EarlGrey/Action/GREYActions.m

```diff
@@ -261,6 +254,69 @@ @implementation GREYActions
 
 #pragma mark - Private
 
+
+/**
+ *  Get WebView input text value.
+ *  @param element The element to target
+ */
++ (NSString *)grey_webviewInputValue:(id)element {
+  return [GREYActions grey_webviewInputValue:element text:@""];
+}
+
+/**
+ *  Sets WebView input text value and returns that value as a string.
+ *  @param element The element to target
+ *  @param text The text to set
+ */
++ (NSString *)grey_webviewInputValue:(id)element
+                                text:(NSString *)text {
```

Comment:
nit: rename to something along grey_webviewSetText:(NSString *)test inInputElement:(id)element, the current name does not make it clear that we are setting the given text.